### PR TITLE
Join all the instructions to enable logs in Jenkins

### PR DESCRIPTION
**Pull request by fermayo**
### What does this PR do?

Puts together all instructions to get logs enabled on the Jenkins plugin under one section.

### Motivation

Align with the in-app onboarding.

### Preview

https://docs-staging.datadoghq.com/fermayo/ciapp-update-jenkins-docs/continuous_integration/setup_pipelines/jenkins/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/setup_pipelines/jenkins.md [hunk 1]
```diff
@@ -50,9 +50,13 @@ DATADOG TRACER CONFIGURATION {...}
 
 {{< img src="ci/ci-jenkins-plugin-config.png" alt="Datadog Plugin configuration for Jenkins"  style="width:100%;">}}
 
-## Enable job log collection on the Agent
+## Enable job log collection
 
-Enable log collection in the Agent:
+This is an optional step that enables the collection of job logs. It involves two steps: enabling the job collection port on the Datadog Agent, and enabling job collection on the Datadog Plugin.
+
+### Enable job log collection on the Agent
```
Comment (ruthnaebeck) [inline content/en/continuous_integration/setup_pipelines/jenkins.md:57]:
```suggestion
### Datadog Agent
```

File: content/en/continuous_integration/setup_pipelines/jenkins.md [hunk 2]
```diff
@@ -50,9 +50,13 @@ DATADOG TRACER CONFIGURATION {...}
...
+
+First, the Datadog Agent needs to be configured to open a TCP port to collect logs:
```
Comment (ruthnaebeck) [inline content/en/continuous_integration/setup_pipelines/jenkins.md:59]:
```suggestion
First, enable job log collection on the Datadog Agent by opening a TCP port to collect logs:
```

File: content/en/continuous_integration/setup_pipelines/jenkins.md [hunk 3]
```diff
@@ -70,14 +74,14 @@ logs:
 
 With this setup, the Agent listens in port `10518` for logs.
 
-## Enable job log collection in the plugin
+### Enable job log collection in the plugin
```
Comment (ruthnaebeck) [inline content/en/continuous_integration/setup_pipelines/jenkins.md:77]:
```suggestion
### Datadog Plugin
```

File: content/en/continuous_integration/setup_pipelines/jenkins.md [hunk 4]
```diff
@@ -70,14 +74,14 @@ logs:
...
 
-You can correlate your pipeline logs with the traces generated by CI Visibility.
+Second, enable job log collection on the plugin:
```
Comment (ruthnaebeck) [inline content/en/continuous_integration/setup_pipelines/jenkins.md:79]:
```suggestion
Second, enable job log collection on the Datadog Plugin:
```

**Review by ruthnaebeck — CHANGES_REQUESTED** [review]

**Review by drodriguezhdez — APPROVED** [review]

**Comment by fermayo** [comment]
@ruthnaebeck this is ready to go

**Review by ruthnaebeck — APPROVED** [review]

**Pull request merged**